packages/react-testing/src/mount.ts

```diff
@@ -78,7 +78,7 @@ type CustomMountResult<
   ? Promise<CustomRoot<Props, Context>>
   : CustomRoot<Props, Context>;
 
-class CustomRoot<Props, Context extends object> extends Root<Props> {
+export class CustomRoot<Props, Context extends object> extends Root<Props> {
```

Comment:
if I don't export this I get the following error

```
Exported variable 'mountWithGraphQL' has or is using name 'CustomRoot' from external module "Shopify/quilt/packages/react-testing/src/mount" but cannot be named.
```
Reason why
https://github.com/Microsoft/TypeScript/issues/5711